submissions/vovan-zt/app_friends/style.scss

```diff
@@ -0,0 +1,260 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap');
+
+$main-color: #45a29e;
+$second-color: #66fcf1;
+$third-color:rgb(186, 124, 233);
+
+
+* {
+    font-family: 'Poppins', sans-serif;
+    margin: 0;
+    padding: 0;
+}
+main {
+	background: #1f2833;
+	min-height: 100vh;
+}
+.header {
+	text-align: center;
+	background-color:#0b0c10;
+	color:$main-color;
+}
+.container {
+	display: grid;
+	grid-template-columns: 200px 1fr;
+	gap: 20px;
+	padding: 10px;
+}
+.filter {
+	width: 200px;
+	height: 530px;
+	background-color:rgba(0, 0, 0, 0.7);
+	margin: 0;
+	box-shadow: 3px 3px 10px  #000;
+	position: fixed;
+	border-radius:5px;
+	&__age, &__name , &__sex {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: space-around;
+		align-content: space-around;
+		width: 180px;
+		height: 100px;
+		padding: 10px;
+		color:#45a29e;
+		box-shadow: 3px 3px 10px  #000;
+		border-radius:5px;
+		&-title {
+			width: 100%;
+			text-align: center;
+			font-size: 22px;
+			font-weight: 400;
+			color:#c5c6c7;
+		}	
+		&-up, &-down {
+			width: 50px;
+			height: 50px;
+			border-radius: 100%;
+			cursor: pointer;
+			background-color:$second-color;
+			box-shadow: 3px 3px 5px rgb(41, 40, 40);
+			transition: 0.5s all;
+			&:hover {
+				img {
+					scale: 0.9;
+				}
+			}
+			img {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)